src/controllers/scenario/index.ts

```diff
@@ -41,12 +42,9 @@ export class scenario extends Controller implements IScenario {
   }
   
   @Get('{executable}')
-  public async get(@Path() executable: Executables): Promise<ExamplesResult> {
-    this.log.debug(`attempting to execute ${executable} scenario`)
+  public async get(@Path() executable: Executables , @Query() params: string[]): Promise<HostResponse> {
```

Comment:
does `params` here represent the args passed to the executable?

Reply:
yes! Happy to rename, to be fair was not sure about the name, i think we should call it args. It got stuck in my head  from last week.